gr-analog/lib/CMakeLists.txt

```diff
@@ -100,8 +78,13 @@ list(APPEND analog_libs
 
 add_library(gnuradio-analog SHARED ${analog_sources})
 target_link_libraries(gnuradio-analog ${analog_libs})
+target_include_directories(gnuradio-analog
+  PUBLIC
+  $<INSTALL_INTERFACE:include>
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
+  )
 GR_LIBRARY_FOO(gnuradio-analog)
-add_dependencies(gnuradio-analog analog_generated_swigs gnuradio-filter)
```

Comment:
that should go into a extra commit fixing the usage of non-existent targets (got detected by setting a CMP policy to new.